automation_tools/satellite6/upgrade/capsule.py

```diff
@@ -43,7 +43,9 @@ def satellite6_capsule_setup(sat_host, os_version):
             print('The following environment variable(s) must be '
                   'set: {0}.'.format(', '.join(missing_vars)))
             sys.exit(1)
-        cap_image = os.environ.get('RHEV_CAP_IMAGE')
+        os.environ['RHEV_CAP_IMAGE'] = 'qe-capsule-upgrade-rhel6-fixed-cap{}'
+        cap_image = os.environ.get('RHEV_CAP_IMAGE').format(os.environ.get(
+            'FROM_VERSION').replace('.', ''))
```

Comment:
If we move the population of "FROM_VERSION" to jenkins-config , then we can update this line to,
```
cap_image = os.environ.get('RHEV_CAP_IMAGE')
```
-----------------------------------------
Also, we can get rid of the line, 
```
os.environ['RHEV_CAP_IMAGE'] = 'qe-capsule-upgrade-rhel6-fixed-cap{}'
```